docs/source/sql.rst

```diff
@@ -37,3 +37,16 @@ These are often an order of magnitude faster than the ``Python->SQLAlchemy``
 route when they are available.::
 
     sqlalchemy.Table <- CSV
+
+Enabling Conversions
+--------------------
+
+Odo is dependent on the SQLAlchemy library for sqlite, PostgreSQL, MySQL, Hive,
+and RedShift support.  SQLAlchemy is not installed by default when Odo is
+installed, therefore you must install it if you want to use one of these
+databases.
+
+If SQLAlchemy is not installed and you try using one of the databases, you
+may get the following error::
```

Comment:
If you go the generalization route, you'd probably want to move this to a different file.